process-automation/latest/model/connectors/docusign.md

```diff
@@ -2,103 +2,214 @@
+| targetFolderId | String | *Optional.* The target folder ID to save the document in Content Services. |
+| targetFolderPath | String | *Requires one.* The location path or relative path of the folder to store the signed document in Content Services. For example, a location path: `/app:company_home/app:user_homes/cm:hruser` or a relative path: `/User Homes/hruser`.  |
+| targetFileType | String | *Optional.* The type to set for the signed file, for example `fin:invoice`. |
+| targetFileMetadata | content-metadata | *Optional.* Metadata assigned to the signed document in Content Services. |
+| underscoreMetadata | boolean | *Optional.* If set to `true` the received prefixed properties names contain underscores (_) instead of colons (:), for separating the namespace prefix from the property name. |
+
+The output parameters from `DOWNLOAD_DOCUMENT` are:
+
+| Property | Type | Description |
+|--- | --- |
+| file | file | The file stored in Content Services of the document. |
+
+The following is an example of the POST body for the Activiti REST API `http://{{domain}}/{{applicationName}}-rb/v1/process-instances` endpoint:
+
+```JSON
 {
-"ahr:contract-type": "Full Time",
-"ahr:full-name": "John Doe",
-"ahr:role": "Developer"
+  "processDefinitionKey": "DocuSignProcessTest",
+  "processInstanceName": "processDocuSignTest_Simple",
+  "businessKey": "MyBusinessKey",
+  "variables": {
+  	"recipientEmail" : "test@test.com",
+  	"recipientName" : "AAE Test",
+  	"emailSubject" : "Sign this document",
+    "files" : [
+        {
+  	        "nodeId" : "<node-id-of-document-to-be-signed>"
+        }
+    ],
+  	"outputFileName" : "testFileName.docx",
+  	"nodeFormat" : "docx",
+  	"documentId" : "10",
+  	"signHerePage" : "1",
+  	"signHereX" : "100",
+  	"signHereY" : "200",
+  	"timeout: "3600"
+  },
+  "payloadType":"StartProcessPayload"
 }
 ```
 
-The output parameters from signing a document are:
+In the business process definition, the service task called **docuSignTask** has the implementation attribute configured to use the connector.
 
-| Parameter | Type | Description |
-| --------- | ---- | ----------- |
-| file | File | *Optional.* The signed document available to be mapped to a [variable]({% link process-automation/latest/model/processes/index.md %}#process-variables). |
+```typescript
+<bpmn2:serviceTask id="ServiceTask_1cheezm" name="docuSignTask" implementation="docuSignConnector.SIGNDOCUMENT">
+      <bpmn2:incoming>SequenceFlow_1siaofh</bpmn2:incoming>
+      <bpmn2:outgoing>SequenceFlow_0nmtcso</bpmn2:outgoing>
+</bpmn2:serviceTask>
+```
 
-## Configuration
+No exceptions or errors are thrown by the connector, however all exceptions are caught and logged. The task execution is always successful, and errors will be returned in an error event.
+
+### Events
+
+The DocuSign connector produces events when the DocuSign envelope changes its status, the events are:
+
+* `ENVELOPE_VOIDED`
```

Comment:
Can you include the definition of each event? https://developers.docusign.com/platform/webhooks/connect/event-triggers/

Reply:
Plus 1 on that one

Reply:
Done cheers